blog/content/post/2020-10-19-security-part-2.md

```diff
@@ -0,0 +1,97 @@
+Google is constantly investigating whether we use best security practices such as TLS / SSL, which are used to encrypt traffic and ensure the confidentiality and integrity of data transmission, as well as server and sometimes customer authentication.
+
+The bottom line is that there is no argument that would challenge making security a priority. Disregarding it does not pay off, and the seemingly saved costs may bring about the opposite result than intended.
+
+### How to deal with security challenges?
+
+The first step is the right approach to the software development process.
+
+There are some rules to follow:
+
+- continuous development of your skills through your own development and training within the organization,
+- sharing knowledge about safety in the team,
+- taking responsibility for not releasing an underdeveloped product under time pressure,
+- good understanding of the impact of risk on the business aspects of operations,
+- following external guidelines such a OWASP,
+- primarily treating safety as priority, not a necessary evil
+
+The rules are obvious, but how many cases do we really stick to them? In how many cases are we not looking for the best solutions or we do not pay enough attention to tests due to the race against time?
+
+We have to answer these questions ourselves. Being aware of your own approach is the beginning of the road to trying to combine speed and security when developing software.
+
+### What's next? Know your enemy.
+
+Fortunately, the problems we face are similar for all developers and there are organizations that work to describe most of them and teach others how to protect themselves from outside threats. Therefore, we do not have to reinvent the wheel.
+
+##### OWASP (Open Web Application Security Project)
+
+is a nonprofit foundation that works to improve the security of software. [OWASP page](https://owasp.org/)
+
+This organization publishes a list of the most common errors in web applications. This list was established in 2017 and is based on real data obtained from companies, organizations and professionals involved in security testing.
+
+##### The current list for 2020:
+
+- Injection
+- Broken Authentication
+- Sensitive Data Exposure
+- XML External Entities (XXE)
+- Broken Access control
+- Security misconfigurations
+- Cross Site Scripting (XSS)
+- Insecure Deserialization
+- Using Components with known vulnerabilities
+- Insufficient logging and monitoring
+
+The list is periodically published and importantly, consists only 10 points. This makes it easy to analyze and understand, making it a good start to taking a serious approach to security in your organization.
+
+A detailed description of each issue can be found in this [article](https://sucuri.net/guides/owasp-top-10-security-vulnerabilities-2020/) and on the [official website of the organization](https://owasp.org/www-project-top-ten/)
```

Comment:
I'm not sure if we should redirect traffic to someone's else article. I would stick with the official owasp website instead.